docs/sources/HTTP.md

```diff
@@ -92,7 +88,20 @@ When the clients post data to the Source Vertex, add `Authorization: Bearer tr3q
 
 ```sh
 TOKEN="Bearer tr3qhs321fjglwf1e2e67dfda4tr"
+# Post data from a Pod in the same namespace of the cluster
 curl -kq -X POST -H "Authorization: $TOKEN" -d "hello world" https://http-pipeline-input:8443/vertices/input
+
+# Or post data from your local with port-forwarding
+kubectl port-forward svc/http-pipeline-input 8443
```

Comment:
should we move this out from `auth` section, perhaps add a section before `auth` to talk about how to send messages to HTTP endpoint?